gellish_communicator/Anything.py

```diff
@@ -194,3 +214,43 @@ def __repr__(self):
         return("Idea: {} lh_uid: {} ({}) {} rh_uid: {}".
                format(self.uid, self.lh_obj.uid, self.rel_type.uid,
                       self.phrase_type_uid, self.rh_obj.uid))
+
+
+if __name__ == "__main__":
```

Comment:
Why is there a __main__ method here?

Reply:
The main method is/was meant for testing the module on its own.